cli/command/manifest/fetch.go

```diff
@@ -44,23 +43,21 @@ func getLocalImageManifestData(namedRef reference.Named, transactionID string) (
 		return nil, nil, err
 	}
 
-	// If this is a manifest list, let's check for it locally so a user can see any modifications
-	// he/she has made.
+	// TODO: is this case relevant if transactionID == "" ?
```

Comment:
Ok, this has changed significantly in #5 so that this doesn't really exist anymore.  

This was only a case in inspect, and inspect was making that incorrect assumption about len() == 1 always meaning a single manifest, so now it explicitly calls `GetList()` or `GetManifest()` so there's no call that has to know about both.